### ISSUE TITLE
Version 1.3 OSX 10.12

### ISSUE THREAD

**Issue by Sebi34** [pecuniabanking/pecunia-client#469]
Hallo, die Version stützt nach der Eingabe des Passwortes ab.

Ich verwende Mac OSX 10.12 Beta 8.

Gruß
Sebastian

**Comment by paulflorianhoberg** [comment]
Selbiges hier mit gleichen Versionen. Pecunia ließ sich erstmal nutzen, nach einem erneuten Öffnen stürzt es direkt nach der Kennworteingabe zur Freigabe der Datenbank ab.

**Comment by paulflorianhoberg** [comment]
Für andere ggf. interessant die aktuell in die selbe Problematik laufen und auf Überweisungen und Co angewiesen sind - 1.1.6 (Build 601) läuft unter sierra!

**Comment by cpromotions** [comment]
Hi,

Ich habe das gleiche Problem unter macOS sierra (10.12).
Bei mir erscheint nichtmal das Fenster zur Passworteingabe. Pecunia meldet nur " Datenbank wird geöffnet..." und geht aus.

> Für andere ggf. interessant die aktuell in die selbe Problematik laufen und auf Überweisungen und Co angewiesen sind - 1.1.6 (Build 601) läuft unter sierra!

Hat bei mir leider nicht funktioniert :-(

Viele Grüße
Tobi

**Comment by frake65** [comment]
Hallo,
ich konnte das Problem mittlerweile umgehen, indem ich wie folgt vorgegangen bin:
- Umbenennen des Ordners ~/Library/Containers/de.pecuniabanking.pecunia nach de.pecuniabanking.pecunia.sav
- Löschen von pecunia.app aus "Programme"
- Herunterladen der Pecunia Version 1.1.6 und Installation dieser Version in Programme 
- Aufruf des Pecunia 1.1.6 - Das Programm will neue Bankenkennung abfragen (ist leer) - Abbruch der Abfrage und Beenden des Programms
- Löschen des Ordners ~/Library/Containers/de.pecuniabanking.pecunia/Data
- Kopieren des Ordners Data aus ~/Library/Containers/de.pecuniabanking.pecunia.sav nach ~/Library/Containers/de.pecuniabanking.pecunia
- Aufruf des Programms und ggf. Eingabe des Kennworts - läuft! :)

**Comment by mike-lischke** [comment]
Es gab früher schon mal Probleme mit verschlüsselten Datenbanken, allerding erinnere ich mich nicht mehr, was das eigentliche Problem war. Also ist es mal wieder Zeit die Datenbank unverschlüsselt zu lassen. Sollte eigentlich sicher genug sein, wenn man ein ordentliches Passwort für das Login verwendet.

**Comment by cpromotions** [comment]
Danke für die Anleitung frake65!

Version 1.1.6 startet zwar. Wenn ich jedoch mein Passwort eingeben verschwindet das Dialogfeld nicht und in der Unteren linken Ecke kommt die Meldung "Datenbank wird geöffnet".
Das wars dann auch schon :-( weiter passiert leider nichts :(
<img width="1198" alt="bildschirmfoto 2016-09-26 um 19 58 04" src="https://cloud.githubusercontent.com/assets/10880920/18845956/1a2fcccc-8424-11e6-9fec-cc276fd76d0f.png">

**Comment by Klara2016** [comment]
Hab auch das Problem,
Mac OS Sierra 10.2. und 10.12.1., Pecunia 1.3.005
Nach Eingabe des Passwortes und bestätigen geht nichts mehr. Das Passwortabfrage-Fenster bleibt.

**Comment by bu5hm4nn** [comment]
@frake65, Der Tipp funktioniert, und zwar auch mit v 1.3!
- Ich habe v 1.1.6 nicht installiert sondern direkt aus dem Download-Image ausgeführt und v1.3 drauf gelassen
- (Da ich die Pekuniadatenbank in einem anderen Ordner habe musste ich meine Datenbank nach dem letzten Schritt von @frake65  in den Einstellungen von Pecunia auswählen (Einstellungen -> Speicherort -> Verwenden))
- Nachdem mit v 1.1.6 alles funktioniert hat, habe ich es geschlossen und v 1.3 geöffnet. Tada, funktioniert!

**Comment by ulidoh** [comment]
Hatte das gleiche Problem wie @cpromotions 

Habe es mit einer Erweiterung des Tipps von frake65 wie folgt gelöst, ohne eine andere Version von Pecunia zu installieren.
* Nachdem man das Verzeichnis ~/Library/Containers/de.pecuniabanking.pecunia in ~/Library/Containers/de.pecuniabanking.pecunia.sav umbenannt hat, startet man Pecunia neu und wird danach gefragt, eine neue Bankkennung zu erstellen. Nun wird
das Verzeichnis ~/Library/Containers/de.pecuniabanking.pecunia neu erstellt. 
* An dieser Stelle abbrechen und Pecunia ganz beenden.
* Jetzt kann man das Verzeichnis ~/Library/Containers/**de.pecuniabanking.pecunia.sav**/Data/Library/Application Support/Pecunia/Data öffnen und die beiden Einträge "accounts" und "shared.sqlite" kopieren und in das neue Verzeichnis ~/Library/Containers/**de.pecuniabanking.pecunia**/Data/Library/Application Support/Pecunia/Data kopieren und die neu angelegten Dateien mit den alten Daten überschreiben
* anschließend habe ich noch per Terminal die Benutzerrechte aller Dateien in dem o.a. Verzeichnis auf 777 gesetzt. Ob das nötig war oder nicht, weiss ich nicht. Danach konnte ich Pecunia allerdings wieder wie gewohnt starten.

**Comment by f3I1x** [comment]
Mit der Vorgehensweise von @ulidoh lässt sich Pecunia wieder bei mir starten (10.12.1). Aber es werden keine Änderungen (z. B. Kategorienzuweiseung) gespeichert und sind beim nächsten Start wieder weg.

**Comment by ulidoh** [comment]
Hast Du auch den von mir zuletzt genannten Punkt durchgeführt, also die Rechte der Dateien auf 777 setzen?

**Comment by f3I1x** [comment]
ja, das habe ich gemacht. Vielleicht liegt es daran, dass ich die Daten verschlüsselt ablege.

Ich habe nun man die Verschlüsselung deaktiviert und nun werden die Änderungen dauerhaft gespeichert. Der einzige Punkt ist jetzt noch, dass die Buchungen nicht mehr automatisch als gelesen markieren werden, wenn ich sie anklicke.

**Comment by Klara2016** [comment]
Ich bin auch auf Version 1.1.6 zurückgerudert nach Anleitung von frake65. Vielen Dank!
Allerdings habe ich den neuen Ordner "de.pecuniabanking.pecunia" gleich nach der Installation mit dem alten überschrieben ohne das Programm vorher zu öffnen und das geht auch.

Ich hatte die neue Pecunia-Version eigentlich nur installiert, weil ich bei 1.1.6 nach der HBCI Umstellung der Banken die Einstellungen meines Kontos nicht ändern konnte. Wundert mich, dass es jetzt trotzdem funktioniert.
